cicd/gitlab/dot.gitlab-ci.yml

```diff
@@ -14,17 +14,11 @@
 
 
 # WORKFLOW RULES -- WHEN TO RUN PIPELINES
-
-# workflow:rules -- using template instead
```

Comment:
could you keep first 2 lines of comment?

Reply:
sorry, we don't need line 18 🤦

Reply:
Done

Reply:
thank you :)